src/main/java/com/opentok/PatchProperties.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * OpenTok Java SDK
+ * Copyright (C) 2021 Vonage.
+ * http://www.tokbox.com
+ *
+ * Licensed under The MIT License (MIT). See LICENSE file for more information.
+ */
+package com.opentok;
+
+/**
+ * Defines values for the <code>properties</code> parameter of the
+ * {@link OpenTok#patchArchive(String, PatchProperties)} and
+ * {@link OpenTok#patchBroadcast(String, PatchProperties)} method
+ * 
+ * @see OpenTok#patchArchive(String, PatchProperties) 
+ * @see OpenTok#patchBroadcast(String, PatchProperties)
+ */
+public class PatchProperties {
+    private boolean hasAudio;
+    private boolean hasVideo;
+    private String addStream;
+    private String removeStream;
+
+    PatchProperties(Builder builder) {
+        this.hasAudio = builder.hasAudio;
+        this.hasVideo = builder.hasVideo;
+        this.addStream = builder.addStream;
+        this.removeStream = builder.removeStream;
+    }
+
+    /**
+     * Use this class to create a PatchProperty object.
+     *
+     * @see PatchProperties
+     */
+    public static class Builder {
+        private boolean hasAudio = true;
+        private boolean hasVideo = true;
+        private String addStream;
+        private String removeStream;
+
+        /**
+         * Call this method to set hasAudio property
+         *
+         * @param hasAudio Whether the stream has Audio or not.
+         *
+         * @return the Builder object with the hasAudio setting.
+         */
+        public Builder hasAudio(boolean hasAudio) {
+            this.hasAudio = hasAudio;
+            return this;
+        }
+
+        /**
+         * Call this method to set hasVideo property
+         *
+         * @param hasVideo Whether the stream has Video or not.
+         *
+         * @return the Builder object with the hasVideo setting.
+         */
+        public Builder hasVideo(boolean hasVideo) {
+            this.hasVideo = hasVideo;
+            return this;
+        }
+
+        /**
+         * Call this method to set addStream property
+         *
+         * @param streamID which stream to add to archive.
+         *
+         * @return the Builder object with the addStream setting.
+         */
+        public Builder addStream(String streamID) {
```

Comment:
You can only add one stream per call to the REST method. (Same comment for `removeStream()`.) See https://tokbox.com/developer/rest/#selecting-archive-streams.

Reply:
as per the last comment since `patchArchive` and `patchBroadcast` has been replaces with `addArchiveStream`, `removeArchiveStream`, `addBroadcastStream` and `removeBroadcastStream` this is no longer an issue. No one will be able to use the `PatchProperties` since no function takes in `PatchProperties` as a param.